doc/CODE_QUALITY.md

```diff
@@ -0,0 +1,41 @@
+# Code Quality Tools
+
+The systemd project has a number of code quality tools set up in the source
+tree and on the github infrastructure. Here's an incomprehensive list of the
+available functionality.
+
+1. Use `ninja -C build test` to run the unit tests. Some tests are skipped if
+   no privileges are available, hence consider also running them with `sudo
+   ninja -C build test`.
```

Comment:
Maybe also mention `-Dslow-tests=true` and `-Dtests=unsafe` and manual tests?

Reply:
do we have a list of manual tests?